fastlane/Fastfile

```
@@ -0,0 +1,100 @@
+default_platform(:android)
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+  desc "Firebase Distributions: Staging Debug App"
+  lane :distribute_staging_debug do
+      gradle(
+        task: "assembleStagingDebug"
+      )
+      release = firebase_app_distribution(
```

rubocop (1.69.1)

[warning] Useless assignment to variable - `release`.

(Lint/UselessAssignment)